testsuite/features/secondary/proxy_container_retail_pxeboot.feature

```diff
@@ -0,0 +1,212 @@
+# Copyright (c) 2018-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# Idempotency note:
+# This feature depends on a JeOS image present on the proxy
+# Please make sure that the feature
+#     features/secondary/buildhost_osimage_build_image.feature
+# has been tested previously
+#
+# The scenarios in this feature are skipped:
+# * if there is no proxy ($proxy is nil)
+# * if there is no private network ($private_net is nil)
+# * if there is no PXE boot minion ($pxeboot_mac is nil)
+#
+# This bug prevents using mgr-bootstrap command:
+#  https://bugzilla.suse.com/show_bug.cgi?id=1220864
+
+@containerized_server
+@skip_if_github_validation
+@buildhost
+@proxy
+@private_net
+@pxeboot_minion
+@scope_retail
+Feature: PXE boot a Retail terminal behind a containerized proxy
+  In order to manage my terminals in a Retail context
+  As the system administrator
+  I PXE boot one of the terminals
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Create hardware type group
+    When I follow the left menu "Systems > System Groups"
+    And I follow "Create Group"
+    And I enter "HWTYPE:Intel-Genuine" as "name"
+    And I enter "Terminal hardware type: genuine Intel" as "description"
+    And I click on "Create Group"
+    Then I should see a "System group HWTYPE:Intel-Genuine created." text
+
+  Scenario: Create branch terminals group
+    When I follow the left menu "Systems > System Groups"
+    And I follow "Create Group"
+    And I enter "example" as "name"
+    And I enter "Terminal branch: example.org" as "description"
+    And I click on "Create Group"
+    Then I should see a "System group example created." text
+    # TODO: Re-add this when not using "Foreign" system type for the proxy anymore:
```

Comment:
Same suggestion than above.

Reply:
A bit harder: it's not a separate scenario.

Maybe it would indeed be a good idea to split the scenarios: this is not about creating the group, but about putting the proxy in the group.

Reply:
I was referring to the TODO description

Reply:
Done, I have taken a mix of your description and mine